config/routes.rb

```diff
@@ -33,6 +33,8 @@
   get "/auth/subdomain_redirect", to: "oauth_redirects#subdomain_redirect", as: :subdomain_redirect
   post "/auth/google/redirect", to: "oauth_redirects#google_redirect", as: :google_oauth_redirect
 
+  get "/external/:id", to: "estimates#external_cw_form_redirect", as: :external
```

Comment:
I'd prefer a slightly more expressive name - reading `external_path` in the view file confused me. Whereas `external_cw_form_redirect_path` would tell me everything I needed to know.